### PR TITLE
fix soft sign deprecation warning

### PR DESCRIPTION
**Pull request by kashif**
softsign was moved from theano.sandbox.softsign to theano.tensor.nnet.nnet

### CODE REVIEW

**Comment by fchollet** [comment]
Tests are failing. For the record, tests are grabbing the latest version of Theano...

**Comment by kashif** [comment]
Having a look now.

File: keras/backend/theano_backend.py [hunk 1]
```diff
@@ -3,7 +3,6 @@
 from theano.sandbox.rng_mrg import MRG_RandomStreams as RandomStreams
 from theano.tensor.signal import pool
 from theano.tensor.nnet import conv3d2d
-from theano.sandbox import softsign as T_softsign
```
Comment (fchollet) [inline keras/backend/theano_backend.py]: Please use `try: from theano.tensor.nnet.nnet import softsign`, `except` for backwards compatbility with previous versions of Theano.

**Comment by fchollet** [comment]
LGTM, thanks!

**Comment by kashif** [comment]
i thank you!

**Pull request merged**